<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSkinWidget</class>
 <widget class="QWidget" name="CSkinWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择皮肤</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="listWidget">
     <item>
      <property name="text">
       <string>黑色</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>蓝色</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>灰色</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>深蓝色</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
